cypress/integration/brewery/ScenarioParameters-additional-advanced-tests.spec.js

```diff
@@ -0,0 +1,169 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import 'cypress-file-upload';
+import utils from '../../commons/TestUtils';
+
+import { DATASET, RUN_TEMPLATE } from '../../commons/constants/brewery/TestConstants';
+import { Downloads, Login, Scenarios, ScenarioManager, ScenarioParameters } from '../../commons/actions';
+import { BreweryParameters } from '../../commons/actions/brewery';
+
+Cypress.Keyboard.defaults({
+  keystrokeDelay: 0,
+});
+
+const SCENARIO_DATASET = DATASET.BREWERY_ADT;
+const SCENARIO_RUN_TEMPLATE = RUN_TEMPLATE.BASIC_TYPES;
+
+function forgeScenarioName() {
+  const prefix = 'Scenario with table - ';
+  return `${prefix}${utils.randomStr(7)}`;
+}
+
+describe('Addtional advanced scenario parameters tests', () => {
+  before(() => {
+    Login.login();
+  });
+
+  beforeEach(() => {
+    Login.relogin();
+  });
+
+  const scenarioNamesToDelete = [];
+  after(() => {
+    Downloads.clearDownloadsFolder();
```

Comment:
Clearing the "downloads" folder is only necessary when we download some files during the test.